modules/training_pipeline/tools/inference_run.py

```diff
@@ -55,7 +55,7 @@ def infer(
     model_cache_dir = Path(model_cache_dir) if model_cache_dir else None
 
     inference_config = configs.InferenceConfig.from_yaml(config_file)
-    inference_api = FinQAInferenceAPI.from_config(
+    inference_api = InferenceAPI.from_config(
```

Comment:
Also, please rename the Beam `App` name to something like: `App(name="inference_qa")`